src/components/media_manager/include/media_manager/audio/from_mic_to_file_recorder_thread.h

```diff
@@ -77,6 +85,7 @@ class FromMicToFileRecorderThread : public threads::ThreadDelegate {
   void deinitArgs();
 
   void psleep(void* timeout);
+  std::string create_caps_string();
```

Comment:
Please add a comment to explain what this function is used for.

Reply:
OK I added in commit 5c9b72ecd09ab2f3427fb37ed716340992ecda0d. Let me know if this is sufficient.